sale_automatic_workflow_with_delay/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,9 @@
+This module allows to execute workflows with a different frequence than
```

Comment:
> with a different frequence

In what manner ?


@TDu I'm extremely doubtful on the usage of this module.

IMHO as @grindtildeath said, the functional needs should be explained in a better way. Technically I'm also doubtful about the way it's done especially this : 

![image](https://user-images.githubusercontent.com/19529533/233034323-254ecc28-f7b7-48ab-935d-493e8e40047c.png)

Thanks to clarify

Reply:
Thierry is off so I'll try to answer your questions :)

In short: the validation of the same orders by the std cron (which runs every 1m) should be blocked by the ctx key  `sale_workflow_validate_with_delay=sale_workflow.validate_order_with_delay`. 

The main use case is: some orders from shop or from EDI must be confirmed automatically but some of them might require a delay to leave sales people a bit of time to check and do modification or block their automatic approval.
Usually we combine this w/ an automatic assignation of the workflow (there's another module that we would like to share).

This should be explained in the README indeed.

Regarding *_job: it's not the same thing and I think the explanation above clarifies this (correct me if I'm wrong).

Regarding the implementation: it's a bit tricky I must admit and we can probably make it a bit smoother, maybe by reworking the base module a little bit but I think Thierry tried to not impact on that one.

Reply:
@simahawk Thanks for the explanation, I definitely got confused by the "with delay" wording used in queue job terminology.

IIUC then the goal is mostly to have a second cron running less often than the standard one and that will validate orders that would not be validated by the first one?

Cannot this be achieved by using an extra filter for order validation (e.g. on the creation date) in a  different workflow record without the need of extra modules? Like a workflow without filter on create date for direct validation and another worflow with filter on create date for what is meant to be a "delayed" validation?

Anyway I'd still suggest to find a better name for the module if it is to be kept in the OCA.